internal/test/build.go

```
@@ -43,7 +44,9 @@ func (fb *FakeOktetoBuilder) Run(_ context.Context, opts *types.BuildOptions) er
 	}
 
 	if opts.Tag != "" {
-		fb.Registry.AddImageByOpts(opts)
+		if err := fb.Registry.AddImageByOpts(opts); err != nil {
+			oktetoLog.Infof("error adding image to registry: %s", err)
+		}
```

Since this is for a test, we should return the error here